internal/workload/command_workload.go

```diff
@@ -0,0 +1,303 @@
+				updateInput.ScopeAccountsInput = &workloads.ScopeAccountsInput{AccountIDs: scopeAccountIDs}
+			}
+
+			_, err := nrClient.Workloads.UpdateWorkload(guid, updateInput)
+			if err != nil {
+				log.Fatal(err)
+			}
+
+			log.Info("success")
+		})
+	},
+}
+
+var cmdDuplicate = &cobra.Command{
+	Use:   "duplicate",
+	Short: "Duplicate a New Relic One workload.",
+	Long: `Duplicate a New Relic One workload
+
+The duplicate command targets an existing workload by its entity GUID, and clones
+it to the provided account ID. An optional name can be provided for the new workload.
+If the name isn't specified, the name + ' copy' of the source workload is used to
+compose the new name.
+`,
+	Example: `newrelic workload duplicate --guid 'MjUyMDUyOHxBOE28QVBQTElDQVRDT058MjE1MDM3Nzk1' --accountID 12345678 --name 'New Workload'`,
+	Run: func(cmd *cobra.Command, args []string) {
+		client.WithClient(func(nrClient *newrelic.NewRelic) {
+			var duplicateInput *workloads.DuplicateInput
+			if name != "" {
+				duplicateInput = &workloads.DuplicateInput{
+					Name: &name,
+				}
+			}
+
+			workload, err := nrClient.Workloads.DuplicateWorkload(accountID, guid, duplicateInput)
+			if err != nil {
+				log.Fatal(err)
+			}
+
+			json, err := prettyjson.Marshal(workload)
+			if err != nil {
+				log.Fatal(err)
+			}
+
+			log.Info("success")
+			fmt.Println(string(json))
+		})
+	},
+}
+
+var cmdDelete = &cobra.Command{
+	Use:   "delete",
+	Short: "Delete a New Relic One workload.",
+	Long: `Delete a New Relic One workload
+
+The delete command accepts a workload's entity GUID.
+`,
+	Example: `newrelic workload delete --guid 'MjUyMDUyOHxBOE28QVBQTElDQVRDT058MjE1MDM3Nzk1'`,
+	Run: func(cmd *cobra.Command, args []string) {
+		client.WithClient(func(nrClient *newrelic.NewRelic) {
+			_, err := nrClient.Workloads.DeleteWorkload(guid)
+			if err != nil {
+				log.Fatal(err)
+			}
+
+			log.Info("success")
+		})
+	},
+}
+
+func init() {
+	// Get
+	Command.AddCommand(cmdGet)
+	cmdGet.Flags().IntVarP(&accountID, "accountId", "a", 0, "the New Relic account ID where you want to create the workload")
+	cmdGet.Flags().IntVarP(&id, "id", "i", 0, "the identifier of the workload")
+	err := cmdGet.MarkFlagRequired("accountId")
+	if err != nil {
+		log.Error(err)
+	}
+
+	err = cmdGet.MarkFlagRequired("id")
+	if err != nil {
+		log.Error(err)
+	}
+
+	// List
+	Command.AddCommand(cmdList)
+	cmdList.Flags().IntVarP(&accountID, "accountId", "a", 0, "the New Relic account ID where you want to create the workload")
+	err = cmdList.MarkFlagRequired("accountId")
+	if err != nil {
+		log.Error(err)
+	}
+
+	// Create
+	Command.AddCommand(cmdCreate)
+	cmdCreate.Flags().IntVarP(&accountID, "accountId", "a", 0, "the New Relic account ID where you want to create the workload")
+	cmdCreate.Flags().StringVarP(&name, "name", "n", "", "the name of the workload")
+	cmdCreate.Flags().StringSliceVarP(&entityGUIDs, "entityGuid", "e", []string{}, "the list of entity Guids composing the workload")
```

Comment:
`entityGuids` as the input name? (takes a list)

Reply:
Usage is `--entityGuid abc --entityGuid xyz`, similar to other commands, I just followed the convention here.

Reply:
Delete tags command for reference: https://github.com/newrelic/newrelic-cli/blob/master/internal/entities/command_tags.go#L234

Reply:
Ah, gotcha.

Reply:
Resolved offline.